### PR TITLE
Documentation/feedback jchml

### PR DESCRIPTION
**Pull request by jozefchmelar**
I made a few changes to improve the readability of the document. It'd be nice to discuss it here

### CODE REVIEW

File: articles/TcOpenFramework/application.md [hunk 1]
```diff
@@ -223,12 +231,12 @@ END_IF;
 After task completion, the state of the task will remain in ```Done```, unless:
 
 1. Task's ```Restore``` method is called (task moves to ```Ready``` state).
+
 1. ``` Invoke``` method is **not called** for two or more cycles of its context (that usually means the same as PLC cycle); successive call of ```Invoke``` will switch the task into ```Ready``` and immediately to ```Requested``` state.
-1. If the task is part of complex coordination primitive, the transition between states will bring the task into a ```Ready``` state if that primitive (StateController and derivatives) has this option enabled.
-1. Alternatively, a ```Restore``` of a task can be called in transition methods of complex coordination primitives(```OnStateChange```).
```
Comment (jozefchmelar) [inline articles/TcOpenFramework/application.md:228]: What's the difference between OnStateChange method and `1. Task's ```Restore``` method is called (task moves to ```Ready``` state).` ?

File: articles/TcOpenFramework/application.md [hunk 2]
```diff
@@ -223,12 +231,12 @@ END_IF;
...
 
+1. If the task is part of complex coordination primitive, the transition between states will bring the task into a ```Ready``` state if that primitive (StateController and derivatives) has this option enabled.
```
Comment (jozefchmelar) [inline articles/TcOpenFramework/application.md:237]: I was trying to rewrite this, but without success. Another try :  Every TcoObject of the StateController, or Sequencer will get restored on new step, or state change if `eRestoreMode.AutoRestoreMembers`  is specified.

File: articles/TcOpenFramework/application.md [hunk 3]
```diff
@@ -237,13 +245,16 @@ The task may finish in an ``` Error``` state. In that case, two recovery scenari
 **(TcoComponent : ITcoComponent)**
 
 The ```component``` in TcOpen is a Function Block/class that controls a physical (Robot, Piston, Drive) or virtual (Operator, Warehouse) component. 
-Another way of thinking about this concept is an ```API/Driver``` that allows the consumer to execute and manage a physical or virtual appliance.
 
-![ComponentSchematics](TcoComponent.png)
```
Comment (jozefchmelar) [inline articles/TcOpenFramework/application.md:242]: This diagram is a bit confusing.

**Review by jozefchmelar — COMMENTED** [review]
@PTKu

**Comment by PTKu** [comment]
comments included manually somewhere arround this commit 1036dd1d8f9c9c062be9df4d80dca2865c9f55d5

**Pull request closed without merging**